easydock/make_clean_copy.py

```diff
@@ -18,13 +18,11 @@ def create_clean_db_copy(db_fname, new_db_fname):
     conn.commit()
 
     res = cur.execute('SELECT * FROM setup')
-    removed_colnames = [d[0] for d in res.description if d[0] != 'yaml']
-    remove_col_sql_line = 'UPDATE setup SET '
-    for colnames in removed_colnames:
-        remove_col_sql_line += colnames + ' = NULL, '
-    remove_col_sql_line = remove_col_sql_line[:-2]
+    remove_colnames = [d[0] for d in res.description if d[0] != 'yaml']
+    sql = 'UPDATE setup SET ' +  ','.join(f'{item} = NULL' for item in remove_colnames)
+    sql = sql[:-2]
```

Comment:
Is this line necessary? It seems not, because there will be no trailing symbols. Or am I mistaken?

Reply:
yes, you're right. There is no comma at the end of the string line anymore, so it should be removed